app/settings/categories/categories.controller.js

```diff
@@ -31,6 +33,13 @@ function (
 
     $scope.refreshView = function () {
         TagEndpoint.queryFresh().$promise.then(function (tags) {
+            _.each(tags, function (tag) {
```

Comment:
Does this end up listing the child categories both at the top level and as children too?